cf.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "------ Create database tables ------"
+
+python manage.py migrate --noinput
+python manage.py load_agency_contacts
```

Comment:
Fantastic! That's really great. So easy to forget to run collectstatic otherwise.